toolchains/solidity/core/crates/slither-server/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "slither-server"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+exclude.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+os = "0.1.0"
+serde = "1.0.193"
+serde_derive = "1.0.193"
+serde_json = "1.0.108"
+tokio = { version="1.34.0", features = ["full"] }
+tower-lsp = "0.20.0"
```

Comment:
shouldn't use it as we should use our own lsp implementation to have a single extension

Reply:
well, ask @0xMemoryGrinder about that, he told me it was not usefull anymore and easier to use